postgres/datadog_checks/postgres/postgres.py

```diff
@@ -663,15 +663,16 @@ def _collect_stats(self, key, db, instance_tags, relations, custom_metrics, func
             self.log.error("Connection error: %s" % str(e))
             raise ShouldRestartException
 
-    def _get_service_check_tags(self, host, port, dbname):
+    def _get_service_check_tags(self, host, port, dbname, tags):
         service_check_tags = [
             "host:%s" % host,
             "port:%s" % port,
-            "db:%s" % dbname
+            "db:%s" % dbname,
+            "tags:%s" % tags
```

Comment:
You should extend the `service_check_tags` list with the `tags` list instead of adding it to `"tags"` key. Customer tags from the configuration are on the same level as the integration tags. The behaviour is the same as for the metric's tags.